status/projects.py

```diff
@@ -150,19 +158,28 @@ def project_summary_data(self, row):
                 row.value["contact"] = (
                     ord_det["owner"]["name"] + ": " + ord_det["owner"]["email"]
                 )
+                field_sources["contact"] = (
+                    "LIMS (researcher.email) and Order Portal, formatted by Genomics Status (backend)"
+                )
             elif "fields" in ord_det:
                 if row.value["contact"] == ord_det["fields"]["project_lab_email"]:
                     row.value["contact"] = (
                         ord_det["fields"]["project_lab_name"]
                         + ": "
                         + ord_det["fields"]["project_lab_email"]
                     )
+                    field_sources["contact"] = (
```

Comment:
Could we define it in a single place instead of in 3 places? Maybe with an if
```suggestion
                    if 'contact' in row.value:
                        field_sources["contact"] = (
```

Reply:
I think my reasoning was to keep it close to where the value is set so that it is harder to forget to update it when a definition is changed. But maybe I can actually use these three different cases to show where it was actually taken from in each case?

Reply:
Fixed this by not repeating the exact same thing, but slight variations.